### ISSUE TITLE
[Bug] [Critical] Packaging Fails in Visual Studio

### ISSUE THREAD

**Issue by steviegt6** [rejuvena/collate#1]
The mod packaging task from Collate fails in Visual Studio, but functions fine through the `dotnet` CLI tool and through Rider.

A sample exception:
```
The "PackageModTask" task could not be loaded from the assembly C:\Users\User\.nuget\packages\rejuvena.collate\1.0.1\build\Rejuvena.Collate.dll.
Could not load file or assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
ResourcePatcher C:\Users\User\.nuget\packages\rejuvena.collate\1.0.1\build\Rejuvena.Collate.targets 89
```

**Comment by Tabizzz** [comment]
`The "PackageModTask" task could not be loaded from the assembly C:\Users\User\.nuget\packages\rejuvena.collate\1.0.1\build\Rejuvena.Collate.dll. Could not load file or assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.	ResourcePatcher	C:\Users\User\.nuget\packages\rejuvena.collate\1.0.1\build\Rejuvena.Collate.targets	89`

**Comment by steviegt6** [comment]
Likely an issue with how I package the library, maybe an issue with the fact I'm including `Rejuvena.Collate.deps.json`?

**Comment by steviegt6** [comment]
Likely an issue with the fact I'm including `Rejuvena.Collate.deps.json`.

**Comment by steviegt6** [comment]
I believe this has been resolved in an unreleased build.

**Comment by steviegt6** [comment]
As far as I'm aware, resolved in f48f6df6f1b13a25337ad19faf58ae6e4cdce052 and c2d57be61860f5401f2d40cb0fdc2548c554d839.
